File: samples/micronaut/src/main/java/io/openapiprocessor/samples/PingController.java
/*
 * Copyright 2021 https://github.com/openapi-processor/openapi-processor-samples
 * PDX-License-Identifier: Apache-2.0
 */

package io.openapiprocessor.samples;

import io.micronaut.http.annotation.Controller;
import io.openapiprocessor.api.PingApi;

@Controller
public class PingController implements PingApi {

    @Override
    public String getPing () {
        return "pong";
    }

}

File: samples/spring-mvc-boot-4-packages-kt/src/main/kotlin/io/openapiprocessor/samples/ping/PingController.kt
/*
 * Copyright 2015 https://github.com/openapi-processor/openapi-processor-samples
 * PDX-License-Identifier: Apache-2.0
 */

package io.openapiprocessor.samples.ping

import org.springframework.web.bind.annotation.RestController

@RestController
class PingController: PingApi {
    override fun getPing(): String {
        return "pong!"
    }
}
